psmdb/percona-server-for-mongodb-4.4-template.yml

```diff
@@ -0,0 +1,269 @@
+   
+        sudo rm -rf test/*
+    - shell: |-
+        rm -rf psmdb-misc-scripts
+        git clone ${MISC_SCRIPTS_REPO} --branch ${MISC_SCRIPTS_BRANCH} --depth 1
+
+        bash -x psmdb_builder.sh --builddir=$(pwd)/test --repo=${PSM_REPO} --branch=${PSM_BRANCH} --jenkins_mode=1 --get_sources=1
+    - shell: |-
+        if [ "${RELEASE_TEST}" != "true" ]; then
+            # build binaries
+
+            if [ -n "${CUSTOM_SUITE}" ]; then
+                UNITTESTS=$(echo "${CUSTOM_SUITE}" | grep -c "^unittests" || true)
+                DBTESTS=$(echo "${CUSTOM_SUITE}" | grep -c "^dbtest" || true)
+                INTEGRATIONTESTS=$(echo "${CUSTOM_SUITE}" | grep -c "^integration_tests" || true)
+                BENCHMARKTESTS=$(echo "${CUSTOM_SUITE}" | grep -c "^benchmarks" || true)
+            else
+                UNITTESTS=$(grep -c "^unittests" psmdb-misc-scripts/suite_sets/${SUITE}.txt || true)
+                DBTESTS=$(grep -c "^dbtest" psmdb-misc-scripts/suite_sets/${SUITE}.txt || true)
+                INTEGRATIONTESTS=$(grep -c "^integration_tests" psmdb-misc-scripts/suite_sets/${SUITE}.txt || true)
+                BENCHMARKTESTS=$(grep -c "^benchmarks" psmdb-misc-scripts/suite_sets/${SUITE}.txt || true)
+            fi
+
+            OPT_TARGETS=" "
+            if [ ${UNITTESTS} -gt 0 ]; then
+                OPT_TARGETS="${OPT_TARGETS} unittests"
+            fi
+
+            if [ "${DBTESTS}" -gt 0 ]; then
+                OPT_TARGETS="${OPT_TARGETS} dbtest"
+            fi
+
+            if [ ${INTEGRATIONTESTS} -gt 0 ]; then
+                OPT_TARGETS="${OPT_TARGETS} integration_tests"
+            fi
+
+            if [ "${BENCHMARKTESTS}" -gt 0 ]; then
+                OPT_TARGETS="${OPT_TARGETS} benchmarks"
+            fi
+
+            PSM_TARGETS=$(echo ${OPT_TARGETS} | xargs)
+            TARBALL_SUFFIX=""
+            if [ -f /etc/debian_version ]; then
+                export OS_RELEASE="$(lsb_release -sc)"
+            fi
+
+            if [ -f /etc/redhat-release ]; then
+                export OS_RELEASE="centos$(lsb_release -sr | awk -F'.' '{print $1}')"
+                RHEL=$(rpm --eval %rhel)
+            fi
+
+            ARCH=$(uname -m 2>/dev/null||true)
+            PSMDIR=percona-server-mongodb
+            PSMDIR_ABS=${WORKSPACE}/percona-server-mongodb
+            TOOLSDIR=${PSMDIR}/mongo-tools
+            TOOLSDIR_ABS=${WORKSPACE}/${TOOLSDIR}
+            TOOLS_TAGS="ssl sasl"
+
+            NJOBS=$(grep -c processor /proc/cpuinfo)
+
+            # link PSM dir to /tmp to avoid "argument list too long error"
+            rm -fr /tmp/${PSMDIR}
+            ln -fs ${PSMDIR_ABS} /tmp/${PSMDIR}
+            cd /tmp
+
+            export CFLAGS="${CFLAGS:-} -fno-omit-frame-pointer"
+            export CXXFLAGS="${CFLAGS}"
+            export INSTALLDIR=${WORKSPACE}/install
+            source $WORKSPACE/test/percona-server-mongodb-42.properties
```

Comment:
`            source $WORKSPACE/test/percona-server-mongodb-42.properties
`
should be changed to:
`            source $WORKSPACE/test/percona-server-mongodb-44.properties
`
since in v4.4 branch we use percona-server-mongodb-**44**.properties

Reply:
Thanks, fixed.